run/testing/grpc_server_streaming.e2e_test.go

```diff
@@ -71,15 +72,22 @@ func TestGRPCServerStreamingService(t *testing.T) {
 	}
 
 	recvMsgs := 0
+	recvFailures := 0
 	for {
 		_, err := resp.Recv()
-		if err == io.EOF {
+		if err == nil {
```

Comment:
It's preferred to indent the error flow. So this would become (with the io.EOF check first):

``` golang
_, err := resp.Recv()
if err == io.EOF {
  break
}
if err != nil {
  recvFailures++
  if recvFailures < 5 {
    // ...
  }
  continue
}

recvMsgs++